include/DataflowPass.h

```diff
@@ -71,7 +72,7 @@ class DataflowPass {
 
     // a helper function that checks for parameter annotations on call
     // instructions. returns true if an annotation was found, and false if not.
-    bool handleIfAnnotationExistsForCallInsts(const std::string &fnName,
+    bool handleIfAnnotationExistsForCallInsts(std::string &fnName,
```

Comment:
Why is this no longer `const`? The comment here doesn't give any indication that this method might change any of its parameters. If it might change `fnName`, the comment above needs to indicate that.

Reply:
`const` initially removed out of low desire to annotate all the other places where `const` would need to be added to allow this function to be annotated with `const`. I'll be adding back to `const` here + all other places where `const` is needed for this function to be `const`